wordpress-mobile/WordPress-iOS
# Update WPMedia to version 0.9.0

This update will improve the performance and quality of scrolling the device media library.

To test:
 - Check the insertion of media on the editor
 - Check navigation on media coming from the WordPress library.

Needs review: @frosty
